test/incidence.jl

```diff
@@ -0,0 +1,240 @@
+module _Incidence
+
+using Test
+using DAECompiler
+using DAECompiler: Incidence, IncidenceValue, Linearity, refresh, linear, linear_time_dependent, linear_state_dependent, linear_time_and_state_dependent, nonlinear
+using DAECompiler.Intrinsics
+using Compiler: SSAValue, Const
+using SparseArrays: rowvals, nonzeros
+
+const *ᵢ = Core.Intrinsics.mul_float
+const +ᵢ = Core.Intrinsics.add_float
+const -ᵢ = Core.Intrinsics.sub_float
+
+walk(ex::Expr, inner, outer) = outer(Expr(ex.head, map(inner, ex.args)...))
+walk(ex, inner, outer) = outer(ex)
+
+postwalk(f, ex) = walk(ex, x -> postwalk(f, x), f)
+
+"""
+    @incidence u₁ + u₂ + t
```

Comment:
Fixes https://github.com/JuliaComputing/OldDAECompiler.jl/issues/177 ;)

Reply:
As suggested in that issue, we could uppercase this macro and use it for printing the lattice element.

Reply:
Implemented in https://github.com/JuliaComputing/DAECompiler.jl/pull/41/commits/b1d0e496388a3dcf4347adc2551eb260932c3773 instead as an `@incidence_str` macro, because `∝u` can't be parsed and there doesn't seem to be good alternatives for `∝`.